flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/SideOutputITCase.java

```diff
@@ -542,6 +544,51 @@ public void processElement2(Integer value, Context ctx, Collector<Integer> out)
 		assertEquals(Arrays.asList(1, 2, 3, 4, 5), resultSink.getSortedResult());
 	}
 
+	/**
+	 * Test keyed KeyedCoProcessFunction side output.
+	 */
+	@Test
+	public void testRealKeyedCoProcessFunctionSideOutput() throws Exception {
```

Comment:
I think you can remove the `Real` here and instead prefix the older tests with `Legacy`, just like for the operators.